File: NetAP/App_Code/DataBean/TemplateBean.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

/// <summary>
/// 適用於連接DB資料用(資料多以可存入DB的格式為主)
/// </summary>
public class TemplateBean //請取有意義的名稱+Bean結尾
{
    /// <summary>
    /// 加班人公司ID
    /// </summary>
    public String CompID { get; set; }

    /// <summary>
    /// 加班人ID
    /// </summary>
    public String EmpID { get; set; }

    /// <summary>
    /// 加班人姓名
    /// </summary>
    public String NameN { get; set; }

    /// <summary>
    /// 加班人性別
    /// </summary>
    public String Sex { get; set; }

     /// <summary>
    /// 最後變更者公司ID
    /// </summary>
    public String LastChgComp { get; set; }

     /// <summary>
    /// 最後變更者ID
    /// </summary>
    public String LastChgID { get; set; }

     /// <summary>
    /// 最後變更時間
    /// </summary>
    public String LastChgDate { get; set; }

}

File: OTWebSite/App_Code/DataBean/TemplateBean.vb
Imports Microsoft.VisualBasic

Public Class TemplateBean
    Private _compID As String
    Private _empID As String
    Private _nameN As String
    Private _sex As String
    Private _lastChgComp As String
    Private _lastChgID As String
    Private _lastChgDate As String
    ''' <summary>
    ''' 加班人公司ID
    ''' </summary>
    Public Property CompID As String
        Get
            Return _compID
        End Get
        Set(ByVal value As String)
            _compID = value
        End Set
    End Property
    ''' <summary>
    ''' 加班人ID
    ''' </summary>
    Public Property EmpID As String
        Get
            Return _empID
        End Get
        Set(ByVal value As String)
            _empID = value
        End Set
    End Property
    ''' <summary>
    ''' 加班人姓名
    ''' </summary>
    Public Property NameN As String
        Get
            Return _nameN
        End Get
        Set(ByVal value As String)
            _nameN = value
        End Set
    End Property
    ''' <summary>
    ''' 加班人性別
    ''' </summary>
    Public Property Sex As String
        Get
            Return _sex
        End Get
        Set(ByVal value As String)
            _sex = value
        End Set
    End Property
    ''' <summary>
    ''' 最後變更者公司ID
    ''' </summary>
    Public Property LastChgComp As String
        Get
            Return _lastChgComp
        End Get
        Set(ByVal value As String)
            _lastChgComp = value
        End Set
    End Property
    ''' <summary>
    ''' 最後變更者ID
    ''' </summary>
    Public Property LastChgID As String
        Get
            Return _lastChgID
        End Get
        Set(ByVal value As String)
            _lastChgID = value
        End Set
    End Property
    ''' <summary>
    ''' 最後變更時間
    ''' </summary>
    Public Property LastChgDate As String
        Get
            Return _lastChgDate
        End Get
        Set(ByVal value As String)
            _lastChgDate = value
        End Set
    End Property
End Class
